var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -28,40 +31,44 @@ class PyElephant(PythonPackage):
     version("0.3.0", sha256="747251ccfb5820bdead6391411b5faf205b4ddf3ababaefe865f50b16540cfef")
 
     variant("docs", default=False, description="Install documentation dependencies")
-    variant("pandas", default=False, description="Build with pandas", when="@0.3.0:0.4.1")
     variant(
         "extras", default=False, description="Build with extras for GPFA, ASSET", when="@0.6.4:"
     )
 
     depends_on("py-setuptools", type="build")
-    depends_on("python@3.7:", type=("build", "run"), when="@0.11.0:")
+
+    depends_on("python@3.8:", type=("build", "run"), when="@0.12.0:")
+
+    depends_on("py-neo@0.10.0:", type=("build", "run"), when="@0.11.0:")
+    depends_on("py-neo@0.9.0", type=("build", "run"), when="@0.9.0:0.10.0")
+    depends_on("py-neo@0.8.0", type=("build", "run"), when="@0.6.4:0.8.0")
     depends_on("py-neo@0.3.4:", type=("build", "run"), when="@0.3.0:0.4.1")
+    depends_on("py-numpy@1.19.5:", type=("build", "run"), when="@0.12.0:")
+    depends_on("py-numpy@1.18.1:1.23.5", type=("build", "run"), when="@0.6.4:0.11.2")
```

Comment:
Same here

Reply:
Numpy version 1.24.0 introduced some changes that affected the functionality of some features in elephant.
This caused issues with older versions of elephant that relied on the previous behavior of numpy. 
These issues were addressed in 0.12.0, which fixed the bugs and ensured compatibility with numpy 1.24.0 and later versions.